diagnostic.md

```diff
@@ -54,7 +68,20 @@ How would we reference each of the following categories of elements?
 -   All `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
+```CSS
+  .big {
+  } elements that belong to the _class_ `big`
+
+  #contentPane{
+   } element matching _ID_ `contentPane`
 
+  #important #red {
```

Comment:
Remember these are classes. So it would be a `.` not a `#`. Also, to have the styling applied to elements that have BOTH classes on it, you do `.important.red`